src/content/documentation/develop/ux-guidelines-for-mobile-extensions.md

```diff
@@ -91,9 +90,13 @@ To provide a seamless experience between your extension and the browser, use the
 
 ### Themes
 
-- Use the Protocol Design System's [color palette](https://protocol.mozilla.org/fundamentals/color.html).
+Use the Protocol Design System’s [color palette](https://protocol.mozilla.org/fundamentals/color.html).
 
-- Match Firefox for Android's [color usage](https://www.figma.com/proto/TebIgESfWmQkMcEmwNaYZl/Android?node-id=0%3A498&viewport=-4823%2C213%2C0.5&scaling=min-zoom&hide-ui=1) for light and dark themes.
+<!--
+- Match Firefox for Android’s [color usage](https://www.figma.com/proto/TebIgESfWmQkMcEmwNaYZl/Android?node-id=0%3A498&viewport=-4823%2C213%2C0.5&scaling=min-zoom&hide-ui=1) for light and dark themes. 
+
+(This link requires figma login, any idea where it should go?)
+-->
```

Comment:
Unless we can get feedback from UX in time, given that we are potentially driving new traffic to these pages, having unusable links is not a good look. So, I would suggest we remove and create an issue to revisit.

Reply:
Personally I don't have a strong opinion, but I'd be ok with filing a followup and remove the figma links for now.

Another option would be to capture what is in the figma links in a screenshot and use that instead of the link to figma (so that the extension developer don't need to necessarily have a figma account to access it), but I would still file an issue to gather from the UX team if there is a better place we could link here (one that wouldn't require an account possibly).

Another details that I noticed when I opened the figma url linked here is that the title states that it is deprecated, which would be one more reasons to file an issue and followup by asking to the UX team (deprecated may means that the figma is deprecated as the place to reference, but it may also means that some of the colors suggested in the figma may have changed in the meantime).

A screenshot of the one linked here in the collapsible details below.

<details>

![image](https://github.com/mozilla/extension-workshop/assets/11484/8b8d0868-aef8-4750-a587-ca8798e5be88)

![image](https://github.com/mozilla/extension-workshop/assets/11484/b10bdc62-33f8-490f-81eb-47c3b643dcee)

![image](https://github.com/mozilla/extension-workshop/assets/11484/3cbb79ca-7045-4114-9768-265a88cd8b10)


</details>

Reply:
Thanks for the screen captures @rpl

Reply:
we don't have any public page with the content above so I'd just remove the link for now (and the rest).

Reply:
See https://github.com/mozilla/extension-workshop/issues/1659